doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -330,6 +330,8 @@ Libraries for networking
 * :ref:`lib_nrf_cloud_rest` library:
 
   * Updated to use the :ref:`lib_rest_client` library for REST API calls.
+  * Added :c:func:`nrf_cloud_rest_send_device_message` function that sends a JSON message to nRF Cloud using its SendDeviceMessage endpoint.
```

Comment:
```suggestion
  * Added :c:func:`nrf_cloud_rest_send_device_message` function that sends a JSON message to nRF Cloud using the `SendDeviceMessage` endpoint.
```

Consulted the docs team involved with release note style. API operation/endpoint names like this should be in backticks to match how they appear in the nRF Cloud docs.